docker/plugins.txt

```diff
@@ -21,17 +23,18 @@ display-url-api:2.3.5
 docker-commons:1.17
 docker-workflow:1.26
 durable-task:1.39
-echarts-api:5.1.2-8
+echarts-api:5.1.2-9
 font-awesome-api:5.15.3-4
-git:4.8.1
+git:4.8.2
 git-client:3.9.0
 git-server:1.10
-github:1.33.1
+github:1.34.0
 github-api:1.123
 github-branch-source:2.11.2
+github-oauth:0.34-SNAPSHOT (private-53e10daa-sam)
```

Comment:
I don't remember the details of this now, but we've removed this from the plugin list before because it caused the build to fail: https://github.com/nationalarchives/tdr-jenkins/commit/543e6a548f94de32b9ee08d5c20b6f5ae88be16c

Reply:
Removed